packages/react-client/src/forks/ReactFlightClientConfig.dom-browser-esm.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+export * from 'react-client/src/ReactFlightClientConfigBrowser';
+export * from 'react-server-dom-esm/src/ReactFlightClientConfigESMBundler';
+export * from 'react-dom-bindings/src/shared/ReactFlightClientConfigDOM';
+export const usedWithSSR = false;
```

Comment:
for the browser build `usedWithSSR` should have always been false, but now that we can separate these renderers we can slightly optimize the build with this config difference